backend/swagger/openapi.yaml

```diff
@@ -112,6 +112,53 @@ paths:
         '401':
           description: Niepoprawny albo brak JWT
 
+  /measurements:
+    get:
+      summary: Zwraca wszystkie pomiary dla uzytkownika
```

Comment:
To wszystkie czy nieprzetworzone?

Na razie niech zostanie tak, trzeba przegadać z frontendem

Reply:
Wszystkie, tego nie było w wymaganiach zrobiłem dla siebie do testów, ale stwierdziłem że może sie przyda więc zostawiłem